src/transformState.js

```diff
@@ -4,8 +4,41 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+const removeKey = function(actionObject, object) {
+  const { keysToRemove } = actionObject;
+
+  for (const keyToRemove of keysToRemove) {
+    delete object[keyToRemove];
+  }
+};
+const clearProperties = function(actionObject, object) {
+  for (const key in object) {
+    if (object.hasOwnProperty(key)) {
+      delete object[key];
+    }
+  }
+};
```

Comment:
I would say arrow function syntax looks better here

```suggestion
const removeKey = (actionObject, object) => {
  const { keysToRemove } = actionObject;

  for (const keyToRemove of keysToRemove) {
    delete object[keyToRemove];
  }
};
const clearProperties = (actionObject, object) => {
  for (const key in object) {
    if (object.hasOwnProperty(key)) {
      delete object[key];
    }
  }
};
```

Reply:
changed to arrow funcs